tenant_schemas/middleware.py

```diff
@@ -75,22 +73,28 @@ class SuspiciousTenantMiddleware(TenantMiddleware):
     TENANT_NOT_FOUND_EXCEPTION = DisallowedHost
 
 
-class DefaultSchemaTenantMiddleware(TenantMiddleware):
+class DefaultTenantMiddleware(SuspiciousTenantMiddleware):
     """
-    This middleware makes it possible for you to set a default schema to be
-    selected when no schema could be determined from the hostname.
+    Extend the SuspiciousTenantMiddleware in scenario where you want to
```

Comment:
`Extend the SuspiciousTenantMiddleware`? should this be `DefaultTenantMiddleware`?